lua/telescope/_extensions/dap.lua

```diff
@@ -15,6 +15,20 @@ local pickers    = require'telescope.pickers'
 local sorters    = require'telescope.sorters'
 local previewers = require'telescope.previewers'
 
+local function get_url_buf(url)
+  local buf = -1
+  if url then
+    local scheme = url:match('^([a-z]+)://.*')
+    if scheme then
+      buf = vim.uri_to_bufnr(url)
+    else
+      buf = vim.uri_to_bufnr(vim.uri_from_fname(url))
```

Comment:
This guarantees that a buffer for that scheme exists

Reply:
Thanks for updating this :)